pkg/ck8s/manifests/k8sd-proxy-template.yaml

```diff
@@ -45,8 +45,9 @@ spec:
               name: k8sd-proxy-config
               key: k8sd-port
         args:
-        - TCP4-LISTEN:2380,fork,reuseaddr
-        - TCP4:$(HOSTIP):$(K8SD_PORT)
+        - -t 5
```

Comment:
Can we add a comment here on the error we've faced and what this resolves so we can refer it in the future.